test/hpke_test.c

```diff
@@ -1799,10 +1787,6 @@ static int test_hpke_compressed(void)
                                     NULL, 0, testctx, NULL)))
         goto end;
     /* now get the compressed form public key */
```

Comment:
Is this comment still valid?

Reply:
yes the comment's valid - weirdly though I'd somehow dropped a call to force the format to compressed. Added back now and nothing else seems amiss.

> The example at the end of the pod file needs updating.

Ta. Fixed.